src/pymor/algorithms/timestepping.py

```diff
@@ -26,22 +26,36 @@
 class TimeStepper(ImmutableObject):
     """Interface for time-stepping algorithms.
 
-    Algorithms implementing this interface solve time-dependent problems
+    Algorithms implementing this interface solve time-dependent initial value problems
     of the form ::
 
-        M * d_t u + A(u, mu, t) = F(mu, t).
+        M(mu) * d_t u + A(u, mu, t) = F(mu, t),
+                         u(mu, t_0) = u_0(mu).
 
     Time-steppers used by |InstationaryModel| have to fulfill
     this interface.
     """
 
     @abstractmethod
+    def estimate_time_step_number(self, initial_time, end_time):
```

Comment:
Sounds strange to my ears. Maybe

```suggestion
    def estimate_time_step_count(self, initial_time, end_time):
```